features/metadata.feature

```diff
@@ -35,7 +35,6 @@ Feature: Metadata
     Background:
         Given a metadata starts with an empty object
 
-    @ignore
     Scenario Outline: Adding some data to metadata
```

Comment:
A good idea did not come to mind. I think if we attempt to somehow resolve, for example, create dedicated repository `functional-metadata-1` for testing for `meta set "foo" "Foo"`, and create `functional-metadata-2` for testing for `meta set "foo" 10` in the same way. Is there any other best idea?